projects/bp-graphql/src/operations/getMostLikedPictures.ts

```diff
@@ -0,0 +1,39 @@
+import { Operation, graphql } from '../Operation.js';
+
+export default {
+  document: graphql`
+    query getMostLikedPictures($filters: PictureFiltersInput!, $pagination: PaginationArg!) {
+      pictures(
+        filters: { and: [{ likes: { ne: null } }, $filters] }
```

Comment:
Maybe change the filter to `> 0` likes? Currently, pictures that have `0` likes are shown, but pictures with `null` likes aren't (`0` means someone liked and disliked the picture, `null` means noone ever liked the picture)